src/modals/SettingsModal/WalletsSettingsSection.tsx

```diff
@@ -116,15 +116,25 @@ const WalletItem = ({ walletName, isCurrent, onWalletDelete }: WalletItemProps)
 
   return (
     <WalletItemContainer
+      role="row"
+      tabIndex={0}
       onMouseEnter={() => setIsShowingDeleteButton(true)}
       onMouseLeave={() => setIsShowingDeleteButton(false)}
     >
-      <WalletName>
+      <WalletName role="cell" tabIndex={0} onFocus={() => setIsShowingDeleteButton(true)}>
         {walletName}
         {isCurrent && <CurrentWalletLabel> {t`(current)`}</CurrentWalletLabel>}
       </WalletName>
       {isShowingDeleteButton && (
-        <WalletDeleteButton squared transparent onClick={() => onWalletDelete(walletName)}>
+        <WalletDeleteButton
+          role="button"
```

Comment:
Shouldn't the `Button` itself always have this role, instead of adding it everytime we style it differently?

Reply:
Ah yeah, since the "root" element / component is `motion.button` there's no need